### ISSUE TITLE
Make a video tutorial (do it pls) 

### ISSUE THREAD

**Issue by iD024** [Mekolaos/Aternos-On-Discord#35]
i am getting error when running pip help

**Comment by iD024** [comment]
i done everything you told

**Comment by NikoIsThere** [comment]
i agree,even with pip intalled and di everything,its just dont work.

**Comment by Back-Slash-N** [comment]
yessir please

**Comment by pritam20ps05** [comment]
please once check my project on this https://github.com/pritam20ps05/mcscraper
it works pretty well and works 98% of the time my server personally uses it.
